src/Tribe/Plugins_API.php

```diff
@@ -136,27 +162,28 @@ public function filter_api_result( $result, $action, $args ) {
 	 *
 	 * @since 4.5.3
 	 *
-	 * @param array $product_data
+	 * @param array  $product_data
+	 * @param string $product_slug
 	 *
 	 * @return array
 	 */
-	public function build_product_data( $product_data ) {
+	public function build_product_data( $product_data, $product_slug ) {
 		$defaults = array(
 			'name'                     => null,
 			'slug'                     => null,
 			'version'                  => null,
 			'author'                   => '<a href="http://m.tri.be/19o3">Modern Tribe, Inc.</a>',
 			'author_profile'           => null,
 			'requires'                 => '3.9',
-			'tested'                   => '4.7.5',
-			'rating'                   => null,
+			'tested'                   => $GLOBALS['wp_version'],
```

Comment:
Support only TEC is active and we're injecting PRO, which isn't active: it probably also isn't installed so I'm not going to be able to parse its readme.

I'd be be reluctant to hardcode from within the get_products, method too, because of the potential for things falling out of date depending on the customers update habits but that's something we could consider?